cmd/server/main.go

```diff
@@ -369,6 +371,35 @@ func main() {
 		})
 	}
 
+	// Initialize Customer
+	var customerService customer.CustomerService
+
+	if entClient != nil {
+		var customerRepo customer.Repository
+		customerRepo, err = customerrepository.New(customerrepository.Config{
+			Client: entClient,
+			Logger: logger.WithGroup("customer.postgres"),
+		})
+		if err != nil {
+			logger.Error("failed to initialize customer repository", "error", err)
+			os.Exit(1)
+		}
+
+		customerService, err = customer.NewService(customer.ServiceConfig{
+			Repository: customerRepo,
+		})
+		if err != nil {
+			logger.Error("failed to initialize customer service", "error", err)
+			os.Exit(1)
+		}
+		defer func() {
+			if err = customerService.Close(); err != nil {
+				logger.Error("failed to close customer service", "error", err)
+			}
+		}()
+	}
```

Comment:
Given that we are not doing anything in the Close() (notifications required that for it's background queues), I would just get rid of Close() altogether.